samples/bluetooth/path_loss_monitoring/README.rst

```diff
@@ -0,0 +1,141 @@
+It consists of a Central and a Peripheral device.
+The Central continuously monitors the signal strength and classifies the connection quality using onboard LEDs.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+To run this sample, you must use two compatible development kits, one programmed with the Central application, and the other with the Peripheral application.
+
+Overview
+********
+
+This sample illustrates Bluetooth LE path loss monitoring by calculating the signal degradation between two connected devices.
+The Central application scans for a Peripheral named ``path_loss_peripheral`` and connects to it.
+After a successful connection, the Central measures path loss using the following formula:
+
+.. code-block:: none
+
+	 path loss = TX Power - RSSI
+
+The application defines the following three signal quality zones based on the path loss value:
+
+* Low path loss (strong signal) - **LED0** is lit.
+* Medium path loss - **LED1** is lit.
+* High path loss (weak signal) - **LED2** is lit.
+
+The Peripheral advertises its presence and accepts connections.
+
+User interface
+**************
+
+Central
+-------
+
+* **LED0:** Indicates **Low** path loss (good signal).
+* **LED1:** Indicates **Medium** path loss.
+* **LED2:** Indicates **High** path loss (weak signal).
+
+No buttons are used.
+
+Peripheral
+----------
+
+No direct user interface.
+The device advertises under the name ``path_loss_peripheral``.
+
+Configuration
+*************
+
+|config|
+
+Additional configuration
+========================
+
+Check and configure the following Kconfig options:
+
+* :kconfig:option:`CONFIG_BT_CENTRAL` - Enables the Bluetooth LE Central role (Central application).
+* :kconfig:option:`CONFIG_BT_PERIPHERAL` - Enables the Bluetooth LE Peripheral role (Peripheral application).
+* :kconfig:option:`CONFIG_BT_DEVICE_NAME` - Sets the advertising name of the Peripheral.
+* :kconfig:option:`CONFIG_BT_SCAN` - Enables scanning for Bluetooth LE advertisements (Central only).
+* :kconfig:option:`CONFIG_BT_SCAN_FILTER_ENABLE` - Enables name-based filtering for scan results (Central only).
+* :kconfig:option:`CONFIG_BT_PATH_LOSS_MONITORING` - Enables the Bluetooth LE path loss monitoring feature on both devices.
+* :kconfig:option:`CONFIG_DK_LIBRARY` - Enables the LEDs on the development kits.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/path_loss_monitoring`
+
+.. include:: /includes/build_and_run.txt
+
+.. |sample_or_app| replace:: sample
+
+.. |ipc_radio_dir| replace:: :file:`sysbuild/ipc_radio`
+
+.. include:: /includes/ipc_radio_conf.txt
+
+Testing
+=======
+
+|test_sample|
+
+1. Program one DK with the :file:`samples/bluetooth/path_loss_monitoring/central` program.
+#. Program another DK with the :file:`samples/bluetooth/path_loss_monitoring/peripheral` program.
```

Comment:
```suggestion
1. Program one DK with the :file:`samples/bluetooth/path_loss_monitoring/central` code.
#. Program another DK with the :file:`samples/bluetooth/path_loss_monitoring/peripheral` code.
```
Not sure program is the right term here. Is it code, application or something else?